build.gradle.kts

```diff
@@ -75,8 +76,24 @@ task<com.github.psxpaul.task.ExecFork>("port-forward-mokey") {
 }
 
 tasks {
+    named<org.springframework.boot.gradle.tasks.bundling.BootJar>("bootJar") {
+        dependsOn("copyDocs")
+    }
+
     named<org.springframework.boot.gradle.tasks.run.BootRun>("bootRun") {
         args("--spring.profiles.active=local-ocp04")
         dependsOn("port-forward-mokey")
     }
+
+    named<org.asciidoctor.gradle.jvm.AsciidoctorTask>("asciidoctor") {
+        sourceDir("src/main/asciidoc")
+        setBaseDir("src/test/resources/graphql")
+        logDocuments = true
+    }
+}
+
+task("copyDocs", type = Copy::class) {
```

Comment:
Får noen warning her, skal prøve å løse dem

```
> Task :copyDocs
Execution optimizations have been disabled for task ':copyDocs' to ensure correctness due to the following reasons:
  - Gradle detected a problem with the following location: '/Users/k82814/workspace/kotlin/gobo/build/resources/main/static/auroraapi'. Reason: Task ':bootJarMainClassName' uses this output of task ':copyDocs' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.4/userguide/validation_problems.html#implicit_dependency for more details about this problem.

> Task :jar
Execution optimizations have been disabled for task ':jar' to ensure correctness due to the following reasons:
  - Gradle detected a problem with the following location: '/Users/k82814/workspace/kotlin/gobo/build/resources/main'. Reason: Task ':jar' uses this output of task ':copyDocs' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.4/userguide/validation_problems.html#implicit_dependency for more details about this problem.
```